### PR TITLE
Oracle build from source v2.5.9 kminder

### PR DESCRIPTION
**Pull request by kminder**

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -17,8 +19,8 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go
 ENV RANCHER_METADATA_BRANCH=release-v2.5
 
-RUN apt-get update && \
-    apt-get install -y gcc ca-certificates git wget curl vim less file xz-utils unzip && \
+RUN microdnf update && \
+    microdnf install gcc ca-certificates git wget curl vim less file unzip tar findutils && \
```
Comment (desagar) [inline Dockerfile.dapper:23]: microdnf doesn't need the `-y` to prevent it from asking questions?

File: go.mod [hunk 2]
```diff
@@ -79,6 +79,7 @@ require (
 	github.com/minio/minio-go/v7 v7.0.10
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/mrjones/oauth v0.0.0-20180629183705-f4e24b6d100c
+	github.com/onsi/ginkgo v1.11.0
```
Comment (desagar) [inline go.mod:82]: was this intentional?

File: package/Dockerfile [hunk 3]
```diff
@@ -151,13 +154,16 @@ ENV CATTLE_SERVER_VERSION ${VERSION}
...
+COPY LICENSE README.md THIRD_PARTY_LICENSES.txt /license/
 COPY k3s-airgap-images.tar /var/lib/rancher/k3s/agent/images/
 RUN chmod +x /usr/bin/entrypoint.sh
 RUN chmod +x /usr/bin/kustomize.sh
 
 COPY data.json /var/lib/rancher-data/driver-metadata/
 
-ENV CATTLE_AGENT_IMAGE ${IMAGE_REPO}/rancher-agent:${VERSION}
+# Force use of rancher-agent image built from source for Verrazzano.
+ENV CATTLE_AGENT_IMAGE ghcr.io/verrazzano/rancher-agent:${VERSION}
```
Comment (desagar) [inline package/Dockerfile:166]: do we need to upgrade rancher-agent as well in BFS?

File: Dockerfile.dapper (hunk 1 above)
Comment (desagar) [reply Dockerfile.dapper:23]: looks like it doesn't

File: go.mod (hunk 2 above)
Comment (kminder) [reply go.mod:82]: This is intentional and was required to get attribution-helper to work.

File: package/Dockerfile (hunk 3 above)
Comment (kminder) [reply package/Dockerfile:166]: The jenkins job builds both from the same repo.

**Review by desagar — APPROVED** [review]

**Pull request merged**